testfiles/UnitarySystem_5ZoneWaterLoopHeatPump.rvi

```diff
@@ -0,0 +1,48 @@
+eplusout.eso
+eplusout.csv
+  Site Outdoor Air Drybulb Temperature
+  Zone Air Temperature
+  Zone Air System Sensible Cooling Rate
+  Zone Air System Sensible Heating Rate
+  Plant Supply Side Cooling Demand Rate
+  Plant Supply Side Heating Demand Rate
+  Boiler NaturalGas Rate
+  Boiler Heating Rate
+  Chiller Electricity Rate
+  Chiller Evaporator Cooling Rate
+  Chiller Evaporator Inlet Temperature
+  Chiller Evaporator Outlet Temperature
+  Chiller Evaporator Mass Flow Rate
+  Chiller Condenser Heat Transfer Rate
+  Pump Mass Flow Rate
+  Pump Outlet Temperature
+  Unitary System Requested Sensible Cooling Rate
+  Unitary System Requested Latent Cooling Rate
+  Unitary System Requested Heating Rate
+  Unitary System Fan Part Load Ratio
+  Unitary System Compressor Part Load Ratio
+  Fan Electricity Rate
+  Heating Coil Electricity Rate
+  Cooling Coil Total Cooling Rate
+  Cooling Coil Sensible Cooling Rate
+  Cooling Coil Source Side Heat Transfer Rate
+  Cooling Coil Runtime Fraction
+  Cooling Coil Part Load Ratio
+  Cooling Coil Air Outlet Temperature
+  Cooling Coil Air Outlet Humidity Ratio
+  Cooling Coil Air Inlet Temperature
+  Cooling Coil Air Inlet Humidity Ratio
+  Cooling Coil Source Side Outlet Temperature
+  Cooling Coil Source Side Inlet Temperature
+  Cooling Coil Air Mass Flow Rate
+  Cooling Coil Source Side Mass Flow Rate
+  Heating Coil Heating Energy
+  Heating Coil Heating Rate
+  Heating Coil NaturalGas Rate
+  Fan Electricity Rate
+  Boiler NaturalGas Rate
+  Boiler Heating Rate
+  Boiler Inlet Temperature
+  Boiler Outlet Temperature
+  Boiler Mass Flow Rate
+0
```

Comment:
Add this rvi file to ensure all outputs are the same, due to output variable order changes.